content/docs/user-guide/interactivity/robotics/concepts-and-components-overview.md

```diff
@@ -0,0 +1,107 @@
+- __Core abstractions__
+  - `ROS2FrameComponent`
+  - `ROS2SensorComponent`
+- __Sensors__
+  - `ROS2CameraSensorComponent`
+  - `ROS2GNSSSensorComponent`
+  - `ROS2IMUSensorComponent`
+  - `ROS2LidarSensorComponent`
+  - `ROS2OdometrySensorComponent`
+- __Robot control__
+  - `AckermannControlComponent`
+  - `RigidBodyTwistControlComponent`
+  - `SkidSteeringControlComponent`
+- __Spawner__
+  - `ROS2SpawnerComponent`
+  - `ROS2SpawnPointComponent`
+- __Vehicle dynamics__
+  - `AckermannVehicleModelComponent`
+  - `SkidSteeringModelComponent`
+  - `WheelControllerComponent`
+- __Robot Import (URDF) system component__
+  - `ROS2RobotImporterSystemComponent`
+
+See the [class diagram](/docs/user-guide/interactivity/robotics/class-diagram/) to understand how components are connected.
+
+### Frames
+
+`ROS2FrameComponent` is a representation of an interesting physical part of the robot. It handles the spatio-temporal relationship between this part and other frames of reference. It also encapsulates namespaces, which help to distinguish between different robots and different parts of the robot, such as in the case of multiple identical sensors on one robot.
+
+All Sensors and the Robot Control component require `ROS2FrameComponent`.
+
+### Sensors
+
+Sensors are components deriving from `ROS2SensorComponent`. They acquire data from the simulated environment and publish it to ROS 2 domain.
+
+- Each sensor has a configuration, including one or more Publishers.
+- Sensors publish at a given rate (frequency).
+- Some sensors can be visualized.
+
+If you intend to add your own sensor, it might be useful to look at how sensors already provided within the O3DE ROS2 Gem are implemented.
+
+### Robot Control
+
+The Gem comes with `ROS2RobotControlComponent`, which you can use to move your robot through:
+
+- [Twist](https://github.com/ros2/common_interfaces/blob/master/geometry_msgs/msg/Twist.msg) messages.
+- [AckermannDrive](https://github.com/ros-drivers/ackermann_msgs/blob/master/msg/AckermannDrive.msg)
+  The component subscribes to these command messages on a configured topic. The topic is `cmd_vel` by default, in a namespace as dictated by __ROS2Frame__.
+
+To make use of received command messages, use either `AckermannControlComponent`, `RigidBodyTwistControlComponent`, or `SkidSteeringControlComponent`, depending on the steering type. You can also implement your own control component or use LUA scripting to handle these commands. Unless scripting is used, control components should translate ROS 2 commands to events on `VehicleInputControlBus`. These events will be handled by a [`VehicleModelComponent`](#vehicle-model) if it is present. You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/) to move your robot with Twist messages. `RobotControl` is suitable to use with [ROS 2 navigation stack](https://navigation.ros.org/). It is possible to implement your own control mechanisms with this component.
```

Comment:
In the future, the ROS 2 Documentation should also link to any scripting API that developers can reference, such as the Lua scripts.

```suggestion
To make use of received command messages, use either `AckermannControlComponent`, `RigidBodyTwistControlComponent`, or `SkidSteeringControlComponent`, depending on the steering type. You can also implement your own control component or use Lua scripting to handle these commands. Unless scripting is used, control components should translate ROS 2 commands to events on `VehicleInputControlBus`. These events will be handled by a [`VehicleModelComponent`](#vehicle-model) if it is present. You can use tools such as [rqt_robot_steering](https://index.ros.org/p/rqt_robot_steering/) to move your robot with Twist messages. `RobotControl` is suitable to use with [ROS 2 navigation stack](https://navigation.ros.org/). It is possible to implement your own control mechanisms with this component.
```